tests/ci/docker_images/linux-aarch/amazonlinux-2_clang-7x_sanitizer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM amazonlinux-2-aarch:clang-7x
```

Comment:
Did you consider joining these two into one docker file as well? In theory the regular clang build can use this image and just not pass in the location of the dependencies.

Reply:
Moved to base.